lib/scidata/squad.ex

```diff
@@ -0,0 +1,189 @@
+defmodule Scidata.Squad do
+  @moduledoc """
+  Module for downloading the [SQuAD1.1 dataset](https://rajpurkar.github.io/SQuAD-explorer).
+  """
+
+  require Scidata.Utils
+  alias Scidata.Utils
+
+  @base_url "https://rajpurkar.github.io/SQuAD-explorer/dataset/"
+  @train_dataset_file "train-v1.1.json"
+  @test_dataset_file "dev-v1.1.json"
+
+  @doc """
+  Downloads the SQuAD dataset.
+
+  ## Examples
+
+      iex> Scidata.Squad.download()
+      %{
+        [
+          %{
+            "paragraphs" => [
```

Comment:
@josevalim do you think we should convert all maps in the train and test datasets here to atom maps? It would require a pass through the data but we could be more specific in typespecs for these functions and resulting maps would be slightly easier to work with.

Reply:
I would keep them as strings I think? To reduce the overheard and because I think dataframes support it?